src/styles/components/dialog/image-preview-dialog.css

```
@@ -21,32 +21,49 @@
   border: none;
-  color: var(--text-secondary);
+  width: 36px;
+  height: 36px;
+  background-color: rgba(131 131 131 / 0.639);
+  color: var(--text-primary);
+  text-shadow: 0 0 4px rgba(0 0 0 / 0.5);
+  cursor: pointer;
+  overflow: hidden;
+  font-size: 28px;
 
   transition: color 0.2s ease;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)